tiledb/sm/global_state/global_state.cc

```diff
@@ -70,6 +70,15 @@ Status GlobalState::initialize(Config* config) {
     RETURN_NOT_OK(init_openssl());
     RETURN_NOT_OK(init_libcurl());
 
+#ifdef __linux__
+    // We attempt to find the linux ca cert bundle
+    // This only needs to happen one time, and then we will use the file found
+    // for each s3/rest call as appropriate
+    VFS vfs;
+    vfs.init(config_.vfs_params());
+    cert_file_ = utils::https::find_ca_certs_linux(vfs);
```

Comment:
don't let it slow the PR down, but this seems like something to log if/when that is feasible.

Reply:
@tdenniston do we have a way to log debug messages? I checked and only saw `LOG_ERROR` and `LOG_STATUS`.

Reply:
No we don't -- in general our error reporting and logging needs an overhaul.